README.md

```diff
@@ -36,6 +36,70 @@ over 100,000 metric values per second with less than 0.2% overhead.
 
 # Building the OVIS / LDMS source code
 
+## Pre-built containers
+
+You may skip the building process and use the pre-built containers from the
+following list. Please follow the corresponding links for more information on
+the images.
+- [ovishpc/ldms-samp](https://hub.docker.com/r/ovishpc/ldms-samp):
+  a small image for 'sampler' daemons meant to be deployed on compute nodes.
+- [ovishpc/ldms-agg](https://hub.docker.com/r/ovishpc/ldms-agg):
+  an image for 'aggregator' daemons which also include various storage plugins.
```

Comment:
This should read "An image for 'aggregator' daemons, which also includes various storage plugins."